internal/driver/commands.go

```diff
@@ -129,115 +129,115 @@ var pprofCommands = commands{
 // reported generated by pprof.
 var pprofVariables = variables{
 	// Filename for file-based output formats, stdout by default.
-	"output": &variable{stringKind, "", "", helpText("Output filename for file-based outputs")},
+	"output": &variable{stringKind, []string{""}, "", helpText("Output filename for file-based outputs")},
```

Comment:
No, I understand filtering on regular expression is not harder than doing the exact match. Just wanted to start with something simpler first, but maybe it's irrelevant.

> Do we only want the slashes to work for numeric labels, or for both types of labels?

Both I think, but see below.

> somthing about '-tagfocus=key1=value1,key2=value2' looks weird to me.

With the equal sign I propose for now we'd just allow it at the beginning. And allow comma-separated values for string tags like today. So:
* `-tagfocus 128kb:512kb` accepts a sample iff it has any numeric tag with memory value in the specified range.
* `-tagfocus mytag=128kb:512kb` accepts a sample iff it has a numeric tag "mytag" with memory value in the specified range. There isn't a way to say `-tagfocus mytag=128kb:512kb,16kb:32kb` or `-tagfocus mytag=128kb:512kb,mytag2=128kb:512kb` for now. Just single value for numeric tags.
* `-tagfocus somecoolregex` accepts a sample iff it has any string tag with "tagkey:tagvalue" string matching the specified regexp.
* `-tagfocus mytag:myvalue` accepts a sample iff it has a string tag named "mytag" with value "myvalue" (omitting the fact that anchors need to be added here for full accuracy).
* `-tagfocus mytag=myvalue1,myvalue2` match two tag values.
* `-tagfocus mytag=myvalue` basically a different way to say the above. I am trying to make sure that the old syntax is still valid. Maybe there are better ways?

I expect that we'd drop the matching of string tag against key:value string at some point and would only match the pattern against the value. Alternatively we could only do the "tag=" syntax for the numeric tags and keep the current matching against key:value forever, but I'd rather converge on something more unified between string and numeric tags.